src/main/cli/cli.c

```diff
@@ -4967,20 +4967,26 @@ static void cliRcSmoothing(const char *cmdName, char *cmdline)
                 cliPrintLinef("%d.%03dms", avgRxFrameUs / 1000, avgRxFrameUs % 1000);
             }
         }
-        cliPrintf("# Active input cutoff: %dhz ", rcSmoothingData->inputCutoffFrequency);
-        if (rcSmoothingData->inputCutoffSetting == 0) {
+        cliPrintf("# Active setpoint cutoff: %dhz ", rcSmoothingData->setpointCutoffFrequency);
+        if (rcSmoothingData->setpointCutoffSetting) {
+            cliPrintLine("(manual)");
+        } else {
             cliPrintLine("(auto)");
+        }
+        cliPrintf("# Active FF cutoff: %dhz (", rcSmoothingData->feedforwardCutoffFrequency);
+        if (rcSmoothingData->ffCutoffSetting) {
+            cliPrintLine("manual)");
         } else {
-            cliPrintLine("(manual)");
+            cliPrintLine("auto)");
```

Comment:
```suggestion
            cliPrintLine("(auto)");
```

Reply:
It's intentional. The "missing" `(` is further up

Reply:
How funny.  I just copied what was there.  Should I make both syntax the same?

Reply:
It's done differently for `# Active FF cutoff:` and `# Active setpoint cutoff:`. Feel free to change either one to make it consistent. I think both approaches are fine

Reply:
Agreed with @klutvott123 that this should be done consistently. Reason: The compiler does deduplication of string constants, so it will only store one each of `(manual)` and `(auto)` as long as the constants are identical.

Reply:
Thanks, I've done it consistently now.